src/templates/locale/pt_BR/LC_MESSAGES/django.po

```diff
@@ -842,6 +838,61 @@ msgstr "País"
 msgid "Sub-Municipality"
 msgstr "Sub-Município"
 
```

Comment:
@eduardocorrearaujo I've included the Scores descriptions, could you check it please?